pynhm/hydrology/PRMSCanopy.py

```diff
@@ -135,6 +133,26 @@ def get_variables() -> tuple:
             # i defined metadata for it in a very adhoc way
         )
 
+    @staticmethod
+    def get_init_values() -> dict:
+        """Get canopy inital values
+
+        Returns:
+            dict: inital values for named variables
+        """
+
+        return {
+            "net_rain": zero,
+            "net_snow": zero,
+            "net_ppt": zero,
+            "intcp_stor": zero,  # JLM: this is the only one that was explicitly defined
```

Comment:
intcp_stor is the only one we need here, right?

Reply:
no, you need them all. they were being implicitly set to zero. if you take any of them out you get a variable not defined error.